WordPress/src/main/java/org/wordpress/android/ui/reader/subfilter/SubfilterListItem.kt

```diff
@@ -30,35 +33,46 @@ sealed class SubfilterListItem(val type: ItemType) {
         }
     }
 
-    enum class ItemType {
-        SECTION_TITLE,
-        SITE_ALL,
-        SITE,
-        DIVIDER,
-        TAG
+    enum class ItemType constructor(val value: Int) {
+        SECTION_TITLE(0),
+        SITE_ALL(1),
+        SITE(2),
+        DIVIDER(3),
+        TAG(4);
+
+        companion object {
+            fun fromInt(value: Int): ItemType? = values().firstOrNull { it.value == value }
+        }
     }
 
     data class SectionTitle(override val label: UiString) : SubfilterListItem(SECTION_TITLE)
 
     object Divider : SubfilterListItem(DIVIDER)
 
     data class SiteAll(
-        override val label: UiString,
         override var isSelected: Boolean = false,
-        override val onClickAction: (filter: SubfilterListItem) -> Unit
-    ) : SubfilterListItem(SITE_ALL)
+        override val onClickAction: ((filter: SubfilterListItem) -> Unit)
```

Comment:
I think this additional parenthesis is added by accident :-), the same goes for the Site and Tag classes.

Reply:
Sure 🤦‍♂️! They staied there while playing around with making them nullable 😊. Changed [here](c2838ce62e13f6eb8a80776aa1db1655a434f2f9)